docs/axelar/bridge-tokens-axelar.md

```diff
@@ -0,0 +1,172 @@
+# Bridge Tokens Between Ethereum and XRPL
+
+<embed src="/snippets/_axelar-disclaimer.md" />
+
+This tutorial describes how to transfer tokens between Ethereum `Sepolia` and the XRPL `Testnet`.
+
+
+## Prerequisites
+
+- [Foundry](https://book.getfoundry.sh/getting-started/installation)
+- An RPC provider, such as Alchemy or Infura, with the `SEPOLIA_RPC_URL` environment variable set to a working `Sepolia` RPC URL.
+- Funded wallets on both chains.
+  - **ETH Sepolia Faucet:** [alchemy.com/faucets/ethereum-sepolia](https://www.alchemy.com/faucets/ethereum-sepolia)
+  - **XRPL Testnet Faucet:** [faucet.tequ.dev](https://faucet.tequ.dev/)
+- [XRPL JS Library](https://js.xrpl.org/)
+
+
+## Steps
+
+### Bridge ETH from Ethereum to XRPL
+
```

Comment:
How about this diagram to explain tehe flow at a high level?
![XRPL Axelar WETH](https://github.com/ripple/opensource.ripple.com/assets/6089916/e09682da-1428-447f-b99e-d443768a9f9a)